docs/pages/whats_new.js

```diff
@@ -57,7 +57,16 @@ function Changelog() {
         </Text>
       </Flex>
 
-      <Markdown text={changelogData} type="changelog" />
+      {changelogData ? (
```

Comment:
@AlbertCarreras could you take a look at #1824 ?

Looking at this PR I was wondering if we could avoid the spinner & provide an instant experience to our pinners.